setup.py

```diff
@@ -14,6 +14,23 @@
 
 from setuptools import setup
 
+# Figure out if the system already has a supported Crypto library
+rsa_signer_library = 'M2Crypto>=0.21.1,<=0.26.4'
```

Comment:
this is a no-op, set it to None here to avoid the double mention of the version string